### PR TITLE
add zombienet config for pendulum/foucoco/amplitude runtime with polkadot

### PR DESCRIPTION
**Pull request by RustNinja**
- add zombienet to pendulum repo to test relay-chain with parachain more easily

### CODE REVIEW

File: zombienet/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+chmod +x zombienet-macos
+cp zombienet-macos /usr/local/bin
+
+
+`./zombienet-macos --help`
+
+build polkadot and medulum with
+
+
+cargo build --release
```
Comment (ebma) [inline zombienet/README.md:17]: Can we get some markdown in here?

**Comment by ebma** [comment]
@RustNinja can you clean this up, so we can merge it soon?

File: zombienet/README.md (hunk 1 above)
Comment (RustNinja) [reply zombienet/README.md]: Done

File: zombienet/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+`wget https://github.com/paritytech/zombienet/releases/download/v1.3.30/zombienet-macos `
+
+`chmod +x zombienet-macos`
+
+`./zombienet-macos --help`
+
+Build Polkadot
+
+`cargo build --release`
+
```
Comment (ebma) [inline zombienet/README.md:16]: Maybe we should be a bit more specific here and say that you first have to git clone polkadot, then navigate to it, checkout some version and then cargo build etc

File: zombienet/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Build Foucoco
+
+`cargo build --release`
```
Comment (ebma) [inline zombienet/README.md:19]: And here we could be more specific and say which package needs to be built or which packages can be built. ie. say something like 
```
cargo build --release --package {amplitude/pendulum/foucoco}-runtime
```

File: zombienet/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+Run zombienet
```
Comment (ebma) [inline zombienet/README.md:21]: This should have some text that points out that the `zombienet/config.toml` by default looks for the `pendulum-node` binary. If another network is to be used the config needs to be changed accordingly.

**Review by ebma — CHANGES_REQUESTED** [review]

File: zombienet/README.md (hunk 3 above)
Comment (RustNinja) [reply zombienet/README.md:19]: Oh. you right we should be more specific. 
We should build pendulum node. and not runtime
and the foucoco/pendulum or amplitude you are able to peek specifying corresponding value in zombienet.toml file.
will change the readme.
@ebma thanks.

**Comment by RustNinja** [comment]
> @RustNinja can you clean this up, so we can merge it soon?

I fixed markdown for readme.md
but actually we can close this PR.
because [98-democratization-rollout](https://github.com/pendulum-chain/pendulum/tree/98-democratization-rollout)
is not a branch for development

File: zombienet/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  [[parachains]]
+  ...
+  chain = "foucoco" 
+  ...
+  ```
+- Run zombienet (specify *provider* and path to zombienet *config.toml* file)
+
+`./zombienet-macos spawn --provider native ./zombienet/config.toml`
+
+Usefull link:
```
Comment (ebma) [inline zombienet/README.md:50]:
```suggestion
Useful link:
```

File: zombienet/README.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+**Zombienet for macos**
+
+Download manualy from [zombienet](https://github.com/paritytech/zombienet/releases)
```
Comment (ebma) [inline zombienet/README.md:3]:
```suggestion
Download manually from [zombienet](https://github.com/paritytech/zombienet/releases)
```

File: zombienet/README.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+- clone polkadot
+
+```
+git clone git@github.com:paritytech/polkadot
+cd polkadot
+```
+
+- build polkadot with *testnet* profile or production
+
+`cargo build --profile testnet -p test-parachain-adder-collator` or `cargo build --release`
```
Comment (ebma) [inline zombienet/README.md:28]: What's the `test-parachain-adder-collator` package? Is it just an example? Why does the testnet profile specify that package and the release one does not?

File: zombienet/config.toml [hunk 8]
```diff
@@ -0,0 +1,20 @@
+[relaychain]
+default_command = "../polkadot/target/release/polkadot"
+default_args = [ "-lparachain=debug" ]
+chain = "rococo-local"
+
+  [[relaychain.nodes]]
+  name = "alice"
```
Comment (ebma) [inline zombienet/config.toml:7]: I have `validator = true` set for both `relaychain.nodes` in my config.toml. Is that necessary? Should we add that flag here too?

File: zombienet/README.md (hunk 7 above)
Comment (RustNinja) [reply zombienet/README.md:28]: You right. there is no needs to add this package. just example. 
Removed.

File: zombienet/config.toml (hunk 8 above)
Comment (RustNinja) [reply zombienet/config.toml:7]: It is not mandatory. but i Added to config.toml file.

Comment (RustNinja) [reply zombienet/config.toml:7]: Done.

**Review by ebma — APPROVED** [review]
Thanks for adding the changes @RustNinja. Ready for merge after CI passed 🙏

**Pull request merged**